src/components/AutoComplete/AutoComplete.tsx

```diff
@@ -58,8 +62,10 @@ function AutoComplete({
   }
 
   const onFocus = () => {
-    if (isCloseOnBlur && value && value.length && items.length) {
-      setIsActiveAutoComplete(true)
+    if (isCloseOnBlur && items.length) {
+      const isValuePresent = value && value.length
+
+      if (isSelect || isValuePresent) setIsActiveAutoComplete(true)
     }
   }
 
```

Comment:
```suggestion
const handleActiveAutoComplete = useCallback(() => {
     setIsActiveAutoComplete((prevVal) => !prevVal)
}, [])
```

To avoid repeat code below